clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ClusterController.groovy

```diff
@@ -238,78 +239,10 @@ class ClusterController {
     try {
       return (Summary) sg.invokeMethod("get${summaryType.capitalize()}Summary".toString(), null /* args */)
     } catch (MissingMethodException e) {
-      throw new SummaryNotFoundException(summaryType: summaryType)
+      throw new NotFoundException("Summary not found (type: ${summaryType})")
     }
   }
 
-  @ExceptionHandler
-  @ResponseStatus(HttpStatus.NOT_FOUND)
-  Map handleClusterNotFoundException(AccountClustersNotFoundException ex) {
-    def message = messageSource.getMessage("account.clusters.not.found", [ex.application, ex.account] as String[], "account.clusters.not.found", LocaleContextHolder.locale)
-    [error: "account.clusters.not.found", message: message, status: HttpStatus.NOT_FOUND]
-  }
-
-  @ExceptionHandler
-  @ResponseStatus(HttpStatus.NOT_FOUND)
-  Map handleClusterNotFoundException(ClusterNotFoundException ex) {
-    def message = messageSource.getMessage("cluster.not.found", [ex.cluster] as String[], "cluster.not.found", LocaleContextHolder.locale)
```

Comment:
I can't see us ever internationalizing error messages ... use of `messageSource` seems overly complicated and unnecessary. Pretty sure we don't even get a good error message out of it right now.

Agree?